### ISSUE TITLE
Add com.snowplowanalytics.snowplow.storage/alter_table_command/jsonschema/1-0-0

### ISSUE THREAD

**Issue by chuwy** [snowplow/iglu-central#781]
Ideas for better name will be highly appreciated.

Right now this is required for BigQuery loader, but idea of payload is very generic. Some Transformer-like application encounters new shredded type and sends a command with this payload to some kind of topic, where another application can receive this command and perform  `ALTER TABLE` statement on assigned storage.

Payload should be following:

```json
{
  "schema": "com.snowplowanalytics.snowplow.storage/alter_table_command/jsonschema/1-0-0",
  "data": {
    "schema": "iglu:com.acme/new_event/jsonschema/1-2-3",
    "type": "DERIVED_CONTEXT"
  }
}
```

**Comment by alexanderdean** [comment]
Interesting - the payload doesn't look much like a `alter_table_command` to me. The payload looks more like something like `newly_observed_type`. There's no **command** here like "mutate x to y".

**Comment by chuwy** [comment]
Good point. But in fact there's actually nothing saying that is *new*. It is just a notion of *snowplow self-describing entity*.

**Comment by chuwy** [comment]
Right now we used to call it "shredded type", but in fact it is something like `self_describing_type`

**Comment by alexanderdean** [comment]
Right, I'd be leaning more towards something like:

```json
{
  "schema": "com.snowplowanalytics.snowplow.storage/observed_type/jsonschema/1-0-0",
  "data": {
    "schema": "iglu:com.acme/new_event/jsonschema/1-2-3",
    "location": "DERIVED_CONTEXT"
  }
}
```

**Comment by chuwy** [comment]
I was wondering if `com.snowplowanalytics.snowplow.storage` vendor and `observed_type` are overly specific. Because in fact there's nothing in payload saying that schema was *observed* and that it is going to be used for storage. It's simply a "shredded type", nothing more: schema and column in canonical enriched event. On the other hand, depending on what migration strategy will be chosen we might want to add here more specific information: e.g. window (when it was *observed*).

**Comment by alexanderdean** [comment]
Agree, `com.snowplowanalytics.snowplow` and `shredded_type` are probably better...
